tests/testthat/test_plots.R

```diff
@@ -260,10 +260,75 @@ test_that('plot functions are producing desired output',{
   
   ## Test plot.rmdFilt ---------------------------------------------------------
   
-  mymetab <- edata_transform(omicsData = metab_object, data_scale = "log2")
-  mymetab <- group_designation(omicsData = mymetab, main_effects = "Condition")
-  rmd_results <- rmd_filter(omicsData = mymetab, metrics=c("MAD", "Skewness", "Correlation"))
-  expect_doppelganger_ci("plot.rmdFilt", plot(rmd_results, pvalue_threshold = 0.01, order_by = "Condition"))
+  mypep <- edata_transform(omicsData = pmartRdata::pep_object, data_scale = "log2")
```

Comment:
pmartRdata is under `Suggests` and so the tests don't actually have access to it during most checks, instead we use the 'little test data' loaded at the top of the test.  

Since were only planning to run these locally I think we could add `testthat::skip_if_not_installed("pmartRdata")` to the top of the test to get rid of the error if you wanted to keep using the pmartRdata for your tests.

Reply:
Sounds good! I made this change since the current test objects aren't large enough to test 2 main effects and pair id fun, happy to switch over if we adjust those at some point or if there is one that does that I overlooked?